### PR TITLE
new records, added bibliography, standardised title

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: new/LIT5992Miracle.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <title xml:lang="gez" xml:id="t1">ተአምረ፡ ኢየሱስ፡ ክርስቶስ፡</title>
                 <title xml:lang="gez" corresp="#t1" type="normalized">Taʾammǝra ʾIyasus Krǝstos</title>
                 <title xml:lang="en" corresp="#t1">Miracle of Jesus Christ</title>
+                <title xml:lang="en" xml:id="t2">Miracle of Jesus Christ:  A sick person is healed from demonic possession by reciting a prayer to Jesus Christ</title>
```
Comment (SusanneHummel) [inline new/LIT5992Miracle.xml:11]: Just to be perfect: "Miracle of Jesus" (without "Christ" as you entitled all the other miracles)

File: new/LIT6020MJCrucifixion.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <body>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6020MJCrucifixion" passive="LIT2382Taamme"/>
+                </listRelation>
+            <div type="edition">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BDLaethd20"/>,
+                    without subscription and supplications.</note>
+                
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>ወበዕለተ፡ አርብ፡ ኮነ፡ አመ፡ ይስቅልዎ፡ ለእግዚእ፡ <hi rend="rubric">ኢየሱስ፡</hi> ወእምዝ፡ አምጽኡ፡ ልብሰ፡
```
Comment (SusanneHummel) [inline new/LIT6020MJCrucifixion.xml:62]: Rubrication should not be encoded in the Lit-files (as before)?

**Comment by thea-m** [comment]
Thank you!

**Comment by thea-m** [comment]
@PietroLiuzzo is there any other way to merge this PR other than manually copying all changes in a new one to solve the conflict? Only one file is conflicted, can this be somehow taken out of the PR?

**Comment by PietroLiuzzo** [comment]
I am not sure, but perhaps It would be enough to move this file from /new to /5001-6000 and commit that change to this branch to have the conflict disappear?

**Comment by thea-m** [comment]
Perfect, thank you!

**Pull request merged**